src/main/resources/templates/apidocumentation/apiDocumentationForTag.ftl

```diff
@@ -1,9 +1,167 @@
+<#import '../govuk/accordion.ftl' as govukAccordion>
 
-<@defaultPage pageHeading="${tagName}">
-  <#list operationList as operation>
-    <@govukDetails.details summaryTitle="${operation.getSummary()}">
-      ${operation.getDescription()!operation.getSummary()}
-    </@govukDetails.details>
-  </#list>
-</@defaultPage>
\ No newline at end of file
+<@defaultPage pageHeading="" twoThirdsColumn=false robotsMeta="all">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter">
+      <@apiDocumentationNav/>
+    </div>
+    <div class="govuk-grid-column-three-quarters">
+      <span class="govuk-caption-xl">API endpoints</span>
+      <h1 class="govuk-heading-xl">${tagName}</h1>
+      <p class="govuk-body">
+        These endpoints return data in PDF format for energy labels, PNG or JPEG for arrow images and JSON for errors.
+      </p>
+      <p class="govuk-body">
+        You can import <a href="#" class="govuk-link">the energy label API OpenAPI 3 document</a> into API testing tools
+        like Postman to test all of our API endpoints.
+      </p>
+      <p class="govuk-body">
+        The values you send will be validated. If there are any validation errors, the response body will be a JSON object
+        containing a <code class="els-code els-code--inline">validationErrors</code> array. For example:
+      </p>
+      <p class="govuk-body">
+        <code class="els-code els-code--block">{
+    "timestamp": "2022-02-18T15:28:11.877Z",
+    "status": 400,
+    "error": "Bad request",
+    "message": "There were validation errors",
+    "validationErrors": [
+      "Field 'modelName' has error: Enter a supplier model identification code",
+      "Field 'supplierName' has error: Enter a supplier name or trade mark"
+    ],
+    "path": "/api/v1/path/to/endpoint"
+  }</code>
+      </p>
+
+      <h2 class="govuk-heading-l">Endpoints</h2>
+
+      <div class="govuk-inset-text">
+        The base URL for all paths is <code class="els-code els-code--inline">https://XXXXXXXXXXXX</code>.
+        No authentication is required for any endpoints.
+      </div>
+
+      <@govukAccordion.accordion accordionId="elg-api-operations">
+        <#list operationList as operation>
+          <@govukAccordion.accordionSection sectionHeading="${operation.getSummary()}" sectionHeadingSize="h3">
+            <#if operation.getDescription()?has_content>
+              <p class="govuk-body">
+                ${operation.getDescription()}
+              </p>
+            </#if>
+
+            <h3 class="govuk-heading-m">Path</h3>
+
+            <p class="govuk-body">
+              <code class="els-code els-code--inline">/api/v1/path/to/endpoint</code>
+            </p>
+
+            <h3 class="govuk-heading-m">Body schema</h3>
+
+            <p class="govuk-body">
+              Send a <code class="els-code els-code--inline">POST</code> request to the path above. The body of the request must be a JSON object with the following properties:
```

Comment:
Might be nice to include an example JSON body so people can just copy/paste it to test? Although I guess they could always just import the OpenAPI file to get that, and it would take up a lot of room on the page.
Maybe something to consider if we get time